code/mathematical-algorithms/amicable_numbers/Amicable_Numbers.c

```diff
@@ -0,0 +1,35 @@
+#include<stdio.h>
+int main(){
+    int i,j;
+    //outer loop.
+    for (i=1;i<=2000;i++){
+        int d1=1;
```

Comment:
Your indentation should be tabs ([Style Guide](https://github.com/OpenGenus/cosmos/blob/master/guides/coding_style/c/style.md#indentation))